SPECS/metal3-ip-address-manager/metal3-ip-address-manager.spec

```diff
@@ -0,0 +1,61 @@
+Summary:        IP Address Manager is a Kubernetes Controller that provides IP addresses and manages the allocations of IP subnets for the Cluster API Provider for Metal3.
+Name:           metal3-ip-address-manager
+Version:        0.0.4
```

Comment:
do you need this specific version ? github repo is at 0.1.1 If you don't need this specific version I strongly recommend you upgrade to latest. We're currently in the process of upgrading all spec in main to their latest...